lib/plugins/filter/before_post_render/indented_code_block.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const stripIndent = require('strip-indent');
```

Comment:
`strip-indent` has been removed.

```suggestion
const { stripIndent } = require('hexo-util');
```

Reply:
Thank you for your advice (just I was bothering!)
I updated.